packages/@o3r/design/builders/generate-style/schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "generateStyleSchematicsSchema",
+  "title": "Generate the style from Design Token",
+  "description": "Generate the style from Design Token",
+  "properties": {
+    "variableType": {
+      "type": "string",
+      "enum": [
+        "css",
+        "scss",
+        "sass"
+      ],
+      "description": "Type of the variables to generate for a Design Token."
+    },
+    "variableReferenceType": {
+      "type": "string",
+      "enum": [
+        "css",
+        "scss",
+        "sass"
+      ],
+      "description": "Type of the variables to generate for a Design Token reference."
+    },
+    "designTokenFilePatterns": {
+      "description": "Path patterns to the Design Token JSON files (it supports Node dependency paths).",
+      "oneOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      ]
+    },
+    "output": {
+      "type": "string",
+      "description": "Path to generate the metadata for CMS. The Metadata will be generated only if the file path is specified"
+    },
+    "defaultStyleFile": {
+      "type": "string",
+      "default": "src/theme.scss",
+      "description": "File path to generate the variable if not determined by the specification"
+    },
+    "metadataOutput": {
+      "type": "string",
+      "description": "If specified, all the generated CSS variable will be generated in the given file. Otherwise, the output file will be determined based on the Variable parameters."
```

Comment:
```suggestion
      "description": "If specified, all the generated CSS variables will be generated in the given file. Otherwise, the output file will be determined based on the Variable parameters."
```

Reply:
also general comment, is the capitalization of certain words necessary? (for example Variable, Metadata, Private Variable, etc)

Reply:
nop, it's a very Bad reflex :D

Reply:
is it possible to remove the capital letters in the documentation and tsdoc ?

Reply:
If you find some please let me know, I'll remove them.